icu4c/source/test/perf/localecanperf/runTest.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/sh
+# Copyright (C) 2020 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html#License
```

Comment:
Please remove the `#License` suffix.

(See also ICU-21257 which I created half an hour ago.)

Reply:
Do *not* remove the whole "License & terms" line!

Do remove *only* the `#License` fragment/suffix.

`# License & terms of use: http://www.unicode.org/copyright.html`

Reply:
https://en.wikipedia.org/wiki/URI_fragment

Reply:
Please take another look.